### ISSUE TITLE
duplicate-except throws red-herring on DoesNotExist

### ISSUE THREAD

**Issue by gbataille** [landscapeio/pylint-django#81]
Hi guys, I have the following issue.
I raised it originally to the pylint team who thinks this is actually due to some processing you are doing with the DoesNotExist (I guess for other code checks)!

https://github.com/PyCQA/pylint/issues/1050
### Steps to reproduce
1. in a django project with 2 models
2. create a piece of code like this

```
    try:
        print("foo")
    except Model1.DoesNotExist:
        print("bar")
    except Model2.DoesNotExist:
        print("baz")
```
### Current behavior

This raises a W0705 duplicate-except saying it's catching django.core.exceptions.ObjectDoesNotExist. both those exception (as far as I understand it) are indeed inheriting from ObjectDoesNotExist (common ancestor), but they are not the same class
### Expected behavior

Do not raise this issue on common ancestor
### pylint --version output

pylint 1.5.6,
astroid 1.4.6
Python 2.7.12 (default, Jun 29 2016, 14:05:02)
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)]

**Comment by sergafts** [comment]
I can reproduce this too.

**Comment by gbataille** [comment]
Hey guys, cannot test yet on my full app because of a bug in pylint that prevents us from moving above 1.7.x

But thanks for the fix. I'll let you know if I experience further issues

Cheers
